.release/docker/talaria_spruce.yaml

```diff
@@ -310,11 +310,13 @@ eventMap:
   default: (( grab $CADUCEUS_ENDPOINT || "http://caduceus:6000/api/v3/notify" ))
 
 jwtValidator:
-  keys:
-    Factory:
-      uri: (( grab $THEMIS_ENDPOINT || "http://themis:6500/keys/docker" ))
-    purpose: 0
-    updateInterval: 24h
+  Config:
+    Resolve:
+      Template: (( grab $THEMIS_ENDPOINT || "http://themis:6500/keys/docker" ))
+    Refresh:
+      Sources:
+        - URI: (( grab $THEMIS_ENDPOINT || "http://themis:6500/keys/docker" ))
+    Leeway: 0
```

Comment:
the uri should match this structure `"https://themis-cd.xmidt.comcast.net/keys/{keyID}"`

Reply:
`"http://themis:6500/keys/docker"` probably should be `"http://themis:6500/keys/{keyID}"`

Reply:
within a docker-compose network, you can refer to other processes by container name (e.g. themis, petasos, etc..).  For this particular environment, the key name is "docker" as defined in the dev (docker) yaml file for themis.